bayeux.go

```diff
@@ -298,6 +335,7 @@ func GetSalesforceCredentials(ap AuthenticationParameters) (creds *Credentials,
 
 func (b *Bayeux) Channel(ctx context.Context, out chan MaybeMsg, r string, creds Credentials, channel string) chan MaybeMsg {
 	b.creds = creds
+	b.status = status
```

Comment:
Back to the original question:

Why you can't access the global "status"?  If you need to expose the global state, just expose a static function.

Reply:
Right.